hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -1,6 +1,24 @@
 # How to boot from a USB Mass Storage Device on a Raspberry Pi 3
 This tutorial explains how to boot your Raspberry Pi 3 from a USB mass storage device such as a flash drive or USB hard disk. Be warned that this feature is experimental and may not work with all USB mass storage devices.
 
+## Before Starting
+Check to make sure your SD card partitions are right for this example:
+```
+$ lsblk
+```
+NAME        MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
+mmcblk0     179:0    0  7.4G  0 disk 
+├─mmcblk0p1 179:1    0  2.3G  0 part 
+├─mmcblk0p2 179:2    0    1K  0 part 
+├─mmcblk0p5 179:5    0   32M  0 part /media/pi/SETTINGS
+├─mmcblk0p6 179:6    0   66M  0 part /boot
+└─mmcblk0p7 179:7    0    5G  0 part /
```

Comment:
Please bear with me, things got a little busy but I have question.  It looks to me the default Raspbian partitions would have /boot in mmcblk0p1, and / (root) in mmcblk0p2?  Is that correct?  If so, that would be easy enough to change.  But I would still like to keep a note for those using noobs - I used it mainly because I wanted a quick setup so I could move onto a project I have in mind.